### PR TITLE
Create React App

### PR DESCRIPTION
**Pull request by kaitcham**
In this branch:

- I initialize the project using Create React App
- I add [React Router](https://reactrouter.com/web/guides/quick-start) and set two Routes and Links for the app's navigation
- I create a books page for displaying all books
- I create a categories page 
-  I create an add-book component for adding a book to booklist

### CODE REVIEW

File: src/pages/CategoriesPage.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+const CategoriesPage = () => <div>Categories Page</div>;
```
Comment (medaminedev66) [inline src/pages/CategoriesPage.js:3]: - [ ] The Categories page should display only a button with text "Check status". Kindly add a button to the categories page.

File: src/components/BookList.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+            key={book.id}
+            id={book.id}
+            name={book.name}
+            author={book.author}
+          />
+        ))}
+      </ul>
+    </div>
+  );
+};
```
Comment (medaminedev66) [inline src/components/BookList.js:33]: - [ ] The BookList component should also render a form by using the AddBook component.

**Review by medaminedev66 — CHANGES_REQUESTED** [review]
Hi @kaitcham  😃, :wave:, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- There are no linter errors ✔️
- You have a descriptive PR ✔️

## Required Changes ♻️
- [ ] Pull Request should have a descriptive title telling others what changes are inside. Please update that.
- [ ] Kindly update your Readme so it has more information about your project. Please use the [Microver template](https://github.com/microverseinc/readme-template) as an example.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/BookList.js (hunk 2 above)
Comment (kaitcham) [reply src/components/BookList.js:33]: Thanks for the review @medaminedev66

Comment (elisha2kyakpo1) [reply src/components/BookList.js:33]: - [ ] Kindly fix the changes requested here by the previous reviewer.

File: src/pages/CategoriesPage.js (hunk 1 above)
Comment (elisha2kyakpo1) [reply src/pages/CategoriesPage.js:3]: - [ ] Kindly fix the changes requested here by the previous reviewer.

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]
Hi 👋 @kaitcham, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by bimbolabuari — APPROVED** [review]
Hi @kaitcham, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**